shell/platform/darwin/ios/framework/Info.plist

```diff
@@ -21,10 +21,10 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
```

Comment:
@jmagman I imagine it's ok but just double checking, both #28783 and #28743 touched this line. Neither of which are in the cherry-pick base. Cherry picking this directly is ok?

Reply:
Is https://github.com/flutter/engine/commit/446068891154db4e9d87a10aa5fd85e6802f94f0 in this branch? Otherwise this doesn't need to be cherry-picked.  I can't tell what the parent SHA is from the GitHub UI.

Reply:
Ah, it's branched off 5b81c6d615c395d70f7032763aa8e89f16fb2bdd (that makes sense).  Just cherry pick https://github.com/flutter/engine/pull/28783 then, it's more focused and lower risk.  Thanks @xster!

Reply:
Ah, you're saying #28783 addresses b/200635527 too? Done

Reply:
Exactly, sorry I wasn't clear! 😄